static/scss/main.scss

```diff
@@ -217,48 +219,129 @@ a#skipnav {
 
 #home {
   .hero {
-    @include span-columns(8);
-    @include shift(2);
-
     text-align: center;
     color: white;
-    padding-top: 40px;
+    padding: 40px 0 60px 0;
 
     h2 {
       font-size: $base-font-size * 7;
       font-weight: 300;
     }
     h3 {
+      display: inline-block;
       font-size: $base-font-size * 2;
       font-weight: 100;
       padding-top: 15px;
+      max-width: 550px;
     }
   }
 
-  #results {
-    @include span-columns(8);
-    @include shift(2);
+  #wrapper {
+    @include clearfix();
 
-    padding: 40px 0;
+    background: #fff;
+    clear: both;
+    color: #000;
+
+    h4 {
+      font-size: 2em;
+    }
+  }
+
+  #results {
+    clear: both;
+    margin: 0 auto;
+    max-width: 840px;
+    padding: 40px 40px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `40px` instead of `40px 40px`